models/eprop_readout.cpp

```diff
@@ -313,23 +308,22 @@ nest::eprop_readout::compute_error_signal_cross_entropy_loss( const long& lag )
 void
 nest::eprop_readout::handle( DelayedRateConnectionEvent& e )
 {
-  size_t rport = e.get_rport();
-  const long delay = e.get_delay_steps();
-  long i = 0;
-
-  assert( rport < 3 );
-
-  std::vector< unsigned int >::iterator it = e.begin();
+  const size_t rport = e.get_rport();
+  assert( rport < SUP_RATE_RECEPTOR );
```

Comment:
```suggestion
  assert( rport < SUP_RATE_RECEPTOR );
   assert( e.get_delay_steps() > 0 );
```
I see that the `delay_steps` are not needed in this solution. Would it maybe make sense to put an assert like in the other handle functions?